app/application.py

```diff
@@ -90,10 +90,14 @@ def _request_requires_authentication():
            and request.method != 'OPTIONS'
 
 
-@app.errorhandler(MessageSaveException)
-def handle_save_exception(error):
-    response = jsonify(error.to_dict())
-    response.status_code = error.status_code
+@app.errorhandler(Exception)
+def handle_error(error):
+    if isinstance(error, MessageSaveException) or isinstance(error, RasNotifyException):
```

Comment:
We could split this out into different `@app.errorhandler`s

Reply:
How does this work when one Exception is inherited from another? Does it handle the most specific one?

Reply:
I think it works on the order they are defined so you'd want the least specific one to be at the bottom

Reply:
http://flask.pocoo.org/docs/0.12/errorhandling/
> Changed in version 0.11: Errorhandlers are now prioritized by specificity of the exception classes they are registered for instead of the order they are registered in.